### PR TITLE
Travis integration test

### PR DESCRIPTION
**Pull request by anderspitman**

### CODE REVIEW

**Comment by anderspitman** [comment]
@jh-bate this PR includes a pretty good integration test for dfaker. It starts dockerized mongo, styx, gatekeeper, shoreline, and jellyfish, uses dfaker to generate 2 days of data, then ensures the data is successfully uploaded to jellyfish.

This is based off of #14 so the diff should be a lot smaller once that is merged.

**Comment by jh-bate** [comment]
thanks @anderspitman will take a look early next week

File: .travis.yml [hunk 1]
```diff
@@ -15,12 +17,17 @@ before_install:
   # The next couple lines fix a crash with multiprocessing on Travis and are not specific to using Miniconda
   - sudo rm -rf /dev/shm
   - sudo ln -s /run/shm /dev/shm
+  - git clone https://github.com/anderspitman/tidepool-docker
+  - tidepool-docker/build_containers tidepool-docker anderspitman
```
Comment (jh-bate) [inline .travis.yml:24]: so does your docker container install the latest versions of the required services ??

Comment (anderspitman) [inline .travis.yml:24]: @jh-bate currently yes. It simply clones each github repo. I designed the containers to mimic the behavior from the scripts in the tools repo as closely as possible. I could pin revisions without too much trouble if that is desired.

Comment (jh-bate) [inline .travis.yml:24]: ta - just interested. maybe soon it could bring it (docker container) into the tools repo if thats ok

**Comment by jh-bate** [comment]
on small thing I thought of - could we add the travis badge to the readme. Just so we can see the state of the build

File: .travis.yml (hunk 1 above)
Comment (anderspitman) [inline .travis.yml:24]: Definitely. It's slightly hacked together at the moment, but with a little cleanup I think it would be generally useful.

**Comment by jh-bate** [comment]
LGTM - merging

**Pull request merged**